### PR TITLE
GStatusBar enhancement and hex editor enhancements

### PR DESCRIPTION
**Pull request by xeons**
-  LibGUI: Add very limited multi-label support to GStatusBar 
-  HexEditor: Added a multi-label status bar, and a new copy option.
-  HexEditor: Fixed startup error

### CODE REVIEW

File: Libraries/LibGUI/GStatusBar.cpp [hunk 1]
```diff
@@ -19,7 +19,10 @@ GStatusBar::GStatusBar(GWidget* parent, int panel_count)
     layout()->set_margins({ 2, 2, 2, 2 });
     layout()->set_spacing(2);
 
-    for (auto i = 0; i < panel_count; i++)
+    if(label_count < 1)
```
Comment (awesomekling) [inline Libraries/LibGUI/GStatusBar.cpp]: Missing space here.

File: Applications/HexEditor/HexEditorWidget.h [hunk 2]
```diff
@@ -8,6 +8,7 @@
 #include <LibGUI/GWidget.h>
 #include <LibGUI/GWindow.h>
 
+
```
Comment (awesomekling) [inline Applications/HexEditor/HexEditorWidget.h]: Unnecessary whitespace change in otherwise untouched file.

File: Applications/HexEditor/HexEditor.cpp [hunk 3]
```diff
@@ -116,6 +116,29 @@ bool HexEditor::copy_selected_text_to_clipboard()
     return true;
 }
 
+bool HexEditor::copy_selected_hex_to_clipboard_as_c_code()
+{
+    if (m_selection_start == -1 || m_selection_end == -1 || (m_selection_end - m_selection_start) < 0 || m_buffer.is_empty())
+        return false;
+
+    StringBuilder output_string_builder;
+    output_string_builder.appendf("unsigned char rawData[%d] = {\n", (m_selection_end - m_selection_start) + 1);
```
Comment (awesomekling) [inline Applications/HexEditor/HexEditor.cpp]: s/rawData/raw_data/ :^)

File: Applications/HexEditor/HexEditorWidget.h [hunk 4]
```diff
@@ -8,7 +8,6 @@
 #include <LibGUI/GWidget.h>
 #include <LibGUI/GWindow.h>
 
-
```
Comment (awesomekling) [inline Applications/HexEditor/HexEditorWidget.h]: Unnecessary whitespace change in otherwise untouched file.

**Comment by xeons** [comment]
I cleaned up the commits a bit, since I didn't want a huge commit bundle. I believe I made all of the suggested changes. :-)

**Comment by awesomekling** [comment]
Nice! :)
